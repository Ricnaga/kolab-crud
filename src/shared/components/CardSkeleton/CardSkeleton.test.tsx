import { describe, expect, it, renderWithTheme } from "@vitest-utils";
import { CardSkeleton } from ".";

describe("Component: CardSkeleton", () => {
  it("should render correctly", () => {
    const { container } = renderWithTheme(<CardSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
