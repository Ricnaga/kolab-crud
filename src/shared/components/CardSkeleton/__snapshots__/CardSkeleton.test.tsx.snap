// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component: CardSkeleton > should render correctly 1`] = `
<div>
  <div
    class="css-1f9j1y1"
  >
    <div
      class="chakra-card__root css-1kfp14p"
    >
      <div
        class="chakra-card__header css-aesgey"
      >
        <div
          class="chakra-skeleton css-1eul3x0"
        />
      </div>
      <div
        class="chakra-card__body css-11e8ur2"
      >
        <div
          class="chakra-skeleton css-7ftq5l"
        />
      </div>
    </div>
  </div>
</div>
`;
